css/main.css

```diff
@@ -70,3 +70,16 @@ body {
   color: #fff;
   border-bottom-color: #fff;
 }
+
+/*google-map responsive formatting */
+.google-map {
+  padding-bottom: 50%;
+  position: relative;
+}
+.google-map iframe {
+  height: 100%;
+  width: 100%;
+  left: 0;
+  top: 0;
```

Comment:
I think it might be a good idea to bring the map up higher

Reply:
No that sounds good